src/main/resources/application.yml

```diff
@@ -2,6 +2,10 @@ spring:
   kafka:
     bootstrap-servers: ${BOOTSTRAP_SERVERSL:localhost:9092}
 
+springfox:
+  documentation:
+    auto-startup: false
+
```

Comment:
Do we still need to keep this?

Reply:
It's part of Zahid's fix for the`-Werror` maven compiler build warnings issue which I've replicated to every repository I've worked on. See here for the original https://github.com/companieshouse/insolvency-delta-consumer/commit/eb5f6c9e05b9a59a5b18c4f1a3d6120a433cd07b